src/store/userAccess/userAccessCommon.ts

```diff
@@ -28,6 +28,10 @@ export const gcpUserAccessQuery: UserAccessQuery = {
   type: 'GCP',
 };
 
+export const ibmUserAccessQuery: UserAccessQuery = {
+  type: 'IBM',
```

Comment:
This will probably need to be 'GCP' in order to see the options in the menu correctly